test/TestImport.spec.ts

```diff
@@ -260,32 +263,44 @@ contract("BErc20", async (accounts) => {
         expect(await bETH.balanceOf(a.user1)).to.be.bignumber.equal(cETHBalance);
         expect(await bBAT.balanceOf(a.user1)).to.be.bignumber.equal(cBATBalance);
 
-        if(checkDebtAfter) {
+        if (checkDebtAfter) {
           console.log("checking debt after");
           expect(await bBAT.borrowBalanceCurrent.call(a.user1)).to.be.bignumber.equal(HUNDRED_BAT);
-          expect(await bUSDT.borrowBalanceCurrent.call(a.user1)).to.be.bignumber.equal(FIVE_HUNDRED_USDT);
-        }
-        else console.log("skipping checking debt after");
+          expect(await bUSDT.borrowBalanceCurrent.call(a.user1)).to.be.bignumber.equal(
+            FIVE_HUNDRED_USDT,
+          );
+        } else console.log("skipping checking debt after");
 
         console.log("checking delegate was revoked");
         const avatar = await bProtocol.registry.getAvatar.call(a.user1);
         expect(await bProtocol.registry.delegate(avatar, bImport.address)).to.be.equal(false);
       });
 
-
       it("user should import without fees", async () => {
+        const avatar1 = await bProtocol.registry.newAvatar.call({ from: a.user1 });
```

Comment:
better use `getAvatarOwner`, as this is what we will use in the website.
specially, avatar might already exist, and it is fine.